src/test/java/rx/CompletableTest.java

```diff
@@ -2700,7 +2700,64 @@ public void call(CompletableSubscriber s) {
         
         Assert.assertTrue(name.get().startsWith("RxComputation"));
     }
-    
+
+    @Test(timeout = 1000)
```

Comment:
I belive you don't need timeout here and in the added tests below, code is synchronous